setup.py

```diff
@@ -44,12 +111,13 @@
     author_email="r.h.p.vorderman@lumc.nl",  # A placeholder for now
     long_description=Path("README.rst").read_text(),
     long_description_content_type="text/x-rst",
+    cmdclass={"build_ext": BuildIsalExt},
     license="MIT",
     keywords="isal isa-l compression deflate gzip igzip",
     zip_safe=False,
     packages=find_packages('src'),
     package_dir={'': 'src'},
-    package_data={'isal': ['*.pxd', '*.pyx']},
+    package_data={'isal': ['*.pxd', '*.pyx', os.path.join('isa-l', '*')]},
```

Comment:
I think you should definitely not include the `isa-l` sources in `package_data`. This means that the contents of that directory will be in `python3.x/site-packages/isal/` after you `pip install` the package! You need to include the LICENSE when you link it statically, but that’s it.

If you need files to be part of the sdist (the source tarball), add them to `MANIFEST.in` instead.

Reply:
Thanks. I was not happy with the isa-l source in the binary distribution as well. (Although that would have been necessary if the source was GPL, but it isn't).